intermediate_html_css/grid/using_flexbox_and_grid.md

```diff
@@ -27,12 +27,12 @@ Content or Layout First Design do not force us to use either Flexbox or Grid! Le
 </p>
 <script async src="https://cpwebassets.codepen.io/assets/embed/ei.js"></script>
 
-It works and looks absolutely *beautiful*! But imagine we would like to move these boxes around in the future. Or, for instance, we wanted the third box to stretch in a second row so that the boxes didn't shrink too much to fit in only one. Both of these things would be *possible* in Grid. But if controlling the layout isn't our priority, Flexbox is more intuitive and fit for the task.
+Çalışıyor ve kesinlikle *güzel* gözüküyor! Ama ileride bu kutuları taşımak istediğimizi düşünelim ya da örnek olarak, üçüncü kutunun ikinci bir satırda genişlemesini isteyelim ki kutular bir satıra sığmak için çok küçülmesinler. Bunların ikisi de Grid ile *mümkün*. Ama düzenin kontrolü bizim için öncelikli değilse Flexbox daha sezgisel ve bu işe uygun olacaktır.
 
-### Combining flexbox and grid
-If you have one-dimensional content, Flexbox can make it easier to control how that content is positioned in a Flex container. If, on the other hand, you want to accurately place content on a complex layout in two-dimensions, Grid can be simpler to use.
+### Flexbox ve Grid'i beraber kullanmak
+Eğer tek boyutlu bir içeriğiniz varsa Flexbox bu içeriklerin Flex konteyneri içerisinde nasıl pozisyonlandırılacağı hakkında daha kolay bir kontrol sağlar. Eğer içeriği çift boyutlu bir düzen içerisinde daha kesin olarak yerleştirmek istiyorsanız Grid kullanmak daha kolay olacaktır.
 
-Say you want your overall layout to be a grid, but you want the grid items to act as flex parents. This way, the grid items can be moved around using the precise two-dimensional placement Grid allows for, while also allowing the content inside the grid items to be capable of freely moving around using Flex. Check out this example from CSS-Tricks.
+Diyelim ki genel düzeninizin Grid olmasını ama nesneleriniz flex nesnesi gibi davranmasını istiyorsunuz. Bu sayede, grid nesneleri Grid'in sağladığı yerleştirme ile iki boyutlu düzende daha kesin olarak hareket edecek; aynı zamanda da grid nesnelerinin içerisindeki içeriğin ise Flex ile daha özgür bir şekilde hareket edecektir. CSS-Tricks'deki bu örneği göz at:
 
 <p class="codepen" data-height="300" data-theme-id="dark" data-default-tab="css,result" data-slug-hash="vYeEOxN" data-editable="true" data-user="TheOdinProjectExamples" style="height: 300px; box-sizing: border-box; display: flex; align-items: center; justify-content: center; border: 2px solid; margin: 1em 0; padding: 1em;">
   <span>See the Pen <a href="https://codepen.io/TheOdinProjectExamples/pen/vYeEOxN">
```

Comment:
```suggestion
  <span>TheOdinProject (<a href="https://codepen.io/TheOdinProjectExamples">@TheOdinProjectExamples</a>) tarafından (<a href="https://codepen.io">CodePen'de</a>) hazırlanan <a href="https://codepen.io/TheOdinProjectExamples/pen/vYeEOxN">Combine Grid and Flexbox</a> adlı ingilizce örneğe bakın</span>
```
Bu tarz örnekler için bulunan yönlendirmeleri de çevirmemiz gerekiyor. Bu öneriyi 38-39-40. satırlara orjinal metindeki düzene göre yazmalısın.